### PR TITLE
add: abs math func

### PR DESCRIPTION
**Pull request by Penglq**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by AlexStocks — APPROVED** [review]

File: math/base_funcs.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gxmath
+
+// Abs abs
+func Abs(n int64) int64 {
```
Comment (jjeffcaii) [inline math/base_funcs.go:21]: Consider `AbsInt64`, `AbsInt32`, `AbsInt16`

Comment (Penglq) [reply math/base_funcs.go:21]: 我想枚举太多不是太好，自己用的时候直接强转就行。后面考虑用泛型来做

**Review by georgehao — APPROVED** [review]

File: math/base_funcs.go (hunk 1 above)
Comment (wongoo) [reply math/base_funcs.go:21]: @Penglq can u add a link to introduce the algorithm in code comment?

Comment (Penglq) [reply math/base_funcs.go:21]: @wongoo https://blog.csdn.net/u010649766/article/details/79815729

**Review by dk-lockdown — APPROVED** [review]

**Review by jjeffcaii — APPROVED** [review]
LGTM

File: math/base_funcs.go (hunk 1 above)
Comment (Penglq) [reply math/base_funcs.go]: 已新增 AbsInt64, AbsInt32, AbsInt16, AbsInt8

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**